api-reference/10 UI Components/dxNumberBox/1 Configuration/format.md

```diff
@@ -8,13 +8,115 @@ default: ''
+
+    You can specify the [precision](/Documentation/ApiReference/Common/Object_Structures/format/#precision) property for decimal numbers.
+
+    NumberBox does not support large number formats. If you want to apply these formats, you can implement the [formatter](/api-reference/50%20Common/Object%20Structures/format/formatter.md '/Documentation/ApiReference/Common/Object_Structures/format/#formatter') and [parser](/api-reference/50%20Common/Object%20Structures/format/parser.md '/Documentation/ApiReference/Common/Object_Structures/format/#parser') functions or specify a custom format string.
+
+- Custom format string    
+
+    Use Unicode Locale Data Markup Language (LDML) patterns to specify a custom format string. An LDML pattern consists of wildcard characters and characters displayed as is. The **format** property supports the following wildcard characters:
+
+    <table class="dx-table">
+        <tr>
+        <th>Format character</th>
+        <th>Description</th>
+        </tr>
+        <tr>
+        <td>0</td>
+        <td>A digit. Displays '0' if it is not specified in the UI.</td>
+        </tr>
+        <tr>
+        <td>&#35;</td>
+        <td>
+                A digit or nothing. One symbol represents several integer digits, but only one decimal digit.</br> 
+                For example, "#0.#" represents "123.4", but not "123.45".
+        </td>
+        </tr>
+        <tr>
+        <td>.</td>
+        <td>
+                A decimal separator. </br>
+                Displayed according to the specified locale.
+        </td>
+        </tr>
+        <tr>
+        <td>,</td>
+        <td>
+                A group separator. </br>
+                Displayed according to the specified locale.
+        </td>
+        </tr>
+        <tr>
+        <td>%</td>
+        <td>
+                The percent sign. Multiplies the input value by 100. </br>    
+                If it is enclosed in single quotes ('%'), it only adds this sign to the input value.
+        </td>
+        </tr>
+        <tr>
+        <td>;</td>
+        <td>Separates positive and negative numbers. If there is no explicit negative format, a positive number receives the "-" prefix. </td>
+        </tr>
+        <tr>
+        <td>Other characters</td>
+        <td>
+                Any character. Should be placed only at the format string's beginning or end. </br>
+                You can use the special characters above as well (in single quotation marks).
+        </td>
+        </tr>
+    </table>
+
+        <!-- tab: JavaScript -->
+        const largeNumber = 123456.789;
+
+        // Limit the decimal part to one digit
+        format: "#0.#" // 123456.7
+
+        // Add a group separator
+        format: ",##0.###" // 123,456.789
+        
+    The example below shows how to specify percentages when you work with decimals. It is necessary to add a `#` character at the start of an LDML pattern to avoid digits before `.` to be deleted.
```

Comment:
Again - try to avoid sentences that start with "it is". 

But in this case, I would rather remove the second sentence. It simply explains again what we already said in the table. If you want to preserve all your leading digits, then use the hash mark ('#'). 

```suggestion
    The examples below show different ways to apply percentage formatting to decimals. 
```